service-front/app/src/Common/src/Service/Lpa/AddOlderLpa.php

```diff
@@ -100,18 +101,60 @@ public function __invoke(
             }
         }
 
+        $this->logger->notice(
+            'Successfully matched LPA {uId} for account with Id {id} ',
+            [
+                'event_code' => EventCodes::OLDER_LPA_FOUND,
+                'id'  => $data['identity'],
+                'uId' => $data['reference_number']
+            ]
+        );
+
+        return new OlderLpaApiResponse(OlderLpaApiResponse::FOUND, ($this->parseOlderLpaMatchResponse)($response));
+    }
+
+    public function confirm(
+        string $userToken,
+        int $lpaUid,
+        string $firstnames,
+        string $lastname,
+        DateTimeInterface $dob,
+        string $postcode,
+        bool $forceActivationKey
+    ): OlderLpaApiResponse {
+        $data = [
+            'reference_number'      => $lpaUid,
+            'first_names'           => $firstnames,
+            'last_name'             => $lastname,
+            'dob'                   => $dob->format('Y-m-d'),
+            'postcode'              => $postcode,
+            'force_activation_key'  => $forceActivationKey
+        ];
+
+        $this->apiClient->setUserTokenHeader($userToken);
+
+        try {
+            $response = $this->apiClient->httpPatch('/v1/older-lpa/confirm', $data);
+        } catch (ApiException $apiEx) {
+            switch ($apiEx->getCode()) {
+                default:
+                    // An API exception that we don't want to handle has been caught, pass it up the stack
+                    throw $apiEx;
+            }
+        }
```

Comment:
```suggestion
        $response = $this->apiClient->httpPatch('/v1/older-lpa/confirm', $data);
```

No need to switch on the code if the only option is default and with that gone there is no benefit to catching and throwing just one type of exception.

Reply:
done